### DISCUSSION TITLE
arguments.length와 함수이름.length with default 매개변수

### DISCUSSION THREAD

**Discussion by woose28** [woowacourse-study/2022-book-muscle#72] [빅터]
커링 장인 준찌가 좋은 질문을 해주셔서 여기에 간단히 퀴즈 형식으로 각색해서 공유합니다!

### Q1. 아래 코드의 실행 결과는?
```javascript
function foo(a, b = 1, c) {
    console.log(arguments.length);
}

foo(); // 1-1번
foo(1); // 1-2번
foo(1, 2); // 1-3번
foo(1, 2, 3);  // 1-4번
```

[질문 관련 코멘트 링크](https://github.com/woowacourse-study/2022-book-muscle/pull/64#discussion_r830438895)

### Q2. 아래 코드의 실행 결과는?
```javascript
function foo(a, b = 1, c) {
    console.log(foo.length);
}

foo(); // 2-1번
foo(1); // 2-2번
foo(1, 2); // 2-3번
foo(1, 2, 3); // 2-4번
```

[질문 관련 코멘트 링크](https://github.com/woowacourse-study/2022-book-muscle/pull/64#discussion_r830439042)

한번 쓰으윽 풀어보면 좋을 거 같아요😊

**Comment by wonsss** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length#description
6
